Decred Wallet/Extensions/Dcrlibwallet.swift

```diff
@@ -49,20 +49,68 @@ extension DcrlibwalletHeadersRescanProgressReport {
     }
 }
 
+
```

Comment:
A single empty line space is usually sufficient. Please delete one of these 2 empty lines.